docs/userguide/datafiles.rst

```diff
@@ -18,9 +18,10 @@ e.g.::
     )
 
 This tells setuptools to install any data files it finds in your packages.
-The data files must be specified via the distutils' ``MANIFEST.in`` file.
+The data files must be specified via the distutils' |MANIFEST.in|_ file.
```

Comment:
Do we still want to call this is a `distutils` file?  I think it is confusing to mention `distutils` at all since it is being largely hidden in `setuptools`.